presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -247,8 +249,57 @@ private String indentString(int indent)
         return Strings.repeat("    ", indent);
     }
 
-    private static class AnySymbolReference
-            extends SymbolReference
+    public static SymbolStem symbolStem(String stem)
+    {
+        return new SymbolStem(stem);
+    }
+
+    public static class SymbolStem extends Symbol
+    {
+        private final String stem;
+
+        SymbolStem(String stem)
+        {
+            // For useful output when we print out failure messages.
+            super(stem + "_*");
+            this.stem = stem;
+        }
+
+        @Override
+        public boolean equals(Object o)
+        {
+            if (this == o) {
+                return true;
+            }
+
+            if (o == null) {
+                return false;
+            }
+
+            if (Symbol.class.equals(o.getClass())) {
+                Symbol other = (Symbol) o;
+                String name = other.getName();
+                int endIndex = name.lastIndexOf('_');
+                String otherStem = endIndex == -1 ? name : name.substring(0, endIndex);
+                return stem.equals(otherStem);
+            }
+
+            if (!SymbolStem.class.equals(o.getClass())) {
+                return false;
+            }
+
+            SymbolStem other = (SymbolStem) o;
+            return stem.equals(other.stem);
+        }
+
+        @Override
+        public int hashCode()
+        {
+            return stem.hashCode();
```

Comment:
See here for more information about the issue of comparing the Orderings: https://github.com/Teradata/presto/pull/326/files#diff-0af664c27fd891a04814795944ca138bR65